src/Makefile

```
@@ -4,20 +4,22 @@
 
 COQDOC_AUTO=coqdoc.css
 
-# Select all the .v files in the current directory
-FILES=$(wildcard *.v)
-FILES_HTML=$(addsuffix .html,$(basename $(FILES)))
+# Select all the .v files in the current directory, excluding node_modules/
+FILES := $(shell find . -path './node_modules' -prune -o -name '*.v' -print)
+FILES_HTML := $(FILES:.v=.html)
 
 all: $(FILES_HTML)
 
 %.html: %.v %.glob jscoqdoc
 	./jscoqdoc $<
+	mv $(<F:.v=.html) $@
```

```suggestion
	mv $(<F:.v=.html) $@
```
the CI bugs because it is ask to do `mv X .html X.html`. I think it is due to this line